src/utils/utils.ts

```diff
@@ -203,6 +205,7 @@ import {
   ADOBE_ANALYTICS_DC_PREFIX,
   BASE_URL,
 } from "../config/constants";
+import appConfig from "../../appConfig";
```

Comment:
Can you move this and the other import statement to the top of the file? That's the general convention for importing packages and other files. I know you're following what was already in this file, but it should've been at the top and it was missed before.

Reply:
oh yes, my apologies!  My IDE added that there automatically, sorry 😅